logging-k8s-stdout-otlp-json/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM eclipse-temurin:21-jre
+
+WORKDIR /usr/src/app/
+
+# renovate: datasource=github-releases depName=opentelemetry-java-instrumentation packageName=open-telemetry/opentelemetry-java-instrumentation
+ENV OPENTELEMETRY_JAVA_INSTRUMENTATION_VERSION=v2.10.0
+
+ADD build/libs/*SNAPSHOT.jar ./app.jar
```

Comment:
what do you think about adding the `../gradlew assemble` here, and then I think could remove it from (the global) `run-acceptance-tests.sh` file?